packages/i3blocks-git/i3blocks-git.pacscript

```diff
@@ -0,0 +1,26 @@
+name="i3blocks-git"
+pkgname="i3blocks"
+pkgdir="${STOWDIR}/${name}"
+description="A feed generator for text based status bars"
+url="https://github.com/vivien/${pkgname}.git"
+depends="autoconf gcc make pkgconf"
```

Comment:
These aren't dependencies that this program needs at runtime, these are only needed to compile right?

Reply:
You're right, changing it to `build_depends`